internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -79,7 +79,9 @@ COPY build/_output/bin/[[.ProjectName]] ${OPERATOR}
 COPY bin /usr/local/bin
 COPY library/k8s_status.py /usr/share/ansible/openshift/
 
-RUN /usr/local/bin/user_setup
+RUN chmod -R g+rwx /usr/local/bin && \
+	/usr/local/bin/user_setup
```

Comment:
Nit: Indent this line with space to match the first line in the RUN command.
```suggestion
    /usr/local/bin/user_setup
```

Ditto for other multi-line RUN commands.